relying-party/src/components/Results.jsx

```diff
@@ -0,0 +1,48 @@
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import './Results.css'
+
+export default function Results() {
+    return (
+        <>
+            <div id="results-box">
+                <h2 class='description'>Red Cross</h2>
+
+                <div id='options'>
+                    <FavoriteIcon sx={{ color: 'red', fontSize: 35 }} />
+                    <button id='apply'>Apply</button>
+                </div>
+                <img class='placeholder' src="./images/placeholder.png" alt="placeholder for companies real image logo" />
```

Comment:
When setting the src attribute of an img tag in React, it'll be better to import the image using JS, and then use that variable in its place.

For example:
`import placeholderImage from '../../images/placeholder.png';`
`<img src={placeholderImage} />`